web_ui/packages/smart-tools/src/utils/opencv-loader.ts

```diff
@@ -1,6 +1,20 @@
 // Copyright (C) 2022-2025 Intel Corporation
 // LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
 
-export default async (): Promise<typeof import('../opencv/4.9.0/opencv.js')> => {
-    return await import('../opencv/4.9.0/opencv.js');
+import type OpenCVTypes from 'OpenCVTypes';
```

Comment:
Even though opencv gets cached, each worker needs its own copy on the context of the webworker. Exporting it here on line 20 is fine because it's just the reference